### PR TITLE
add @ads for kugou & ximalaya apps.

### PR DESCRIPTION
**Pull request by norci**

### CODE REVIEW

File: data/category-ads-all [hunk 1]
```diff
@@ -12,6 +12,7 @@ contextual.media.net @ads
 cpmstar.com @ads
 demdex.net @ads
 httpool.com @ads
+kugou@ads
```
Comment (IceCodeNew) [inline data/category-ads-all:15]: that is not how it works.

File: data/category-ads-all [hunk 2]
```diff
@@ -23,6 +24,7 @@ snapads.com @ads
 spotxchange.com @ads
 unimhk.com @ads
 upapi.net @ads
+ximalaya@ads
```
Comment (IceCodeNew) [inline data/category-ads-all:27]: same here

File: data/kugou [hunk 3]
```diff
@@ -1,2 +1,6 @@
 kgimg.com
 kugou.com
+ads.service.kugou.com @ads
```
Comment (IceCodeNew) [inline data/kugou:3]: sort

File: data/ximalaya [hunk 4]
```diff
@@ -1,2 +1,6 @@
 ximalaya.com
 xmcdn.com
+
+adbehavior.ximalaya.com @ads
```
Comment (IceCodeNew) [inline data/ximalaya:4]: sort

**Review by IceCodeNew — CHANGES_REQUESTED** [review]

**Comment by IceCodeNew** [comment]
You are supposed to create files named `ximalaya-ads` & `kugou-ads`, then include them by syntax `include: ximalaya-ads`

**Comment by norci** [comment]
> You are supposed to create files named `ximalaya-ads` & `kugou-ads`, then include them by syntax `include: ximalaya-ads`

done. thanks for you help.

**Review by rootmelo92118 — APPROVED** [review]

**Review by IceCodeNew — APPROVED** [review]

**Pull request merged**